sda/cmd/api/swagger_v1.yml

```diff
@@ -0,0 +1,410 @@
+      parameters:
+        - in: path
+          name: accessionID
+          schema:
+            type: string
+          required: true
+      responses:
+        "200":
+          description: Successful operation.
+        "400":
+          description: Bad payload
+        "401":
+          description: Authentication failure.
+        "500":
+          description: Internal application error.
+  /files:
+    get:
+      description: List all files belonging to the calling user that is not part of a dataset.
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/FileInfo"
+          description: Successful operation
+        "400":
+          description: Bad username
+        "401":
+          description: Authentication failure
+        "500":
+          description: Internal application error
+  /ready:
+    get:
+      description: Returns the status of the application.
+      responses:
+        "200":
+          content:
+            application/json:
+              example:
+                {}
+          description: The service is operational
+        "503":
+          content:
+            application/json:
+              example:
+                {}
+          description: Unhealthy service
+      security: []
+  /users:
+    get:
+      description: Lists all users with ongoing submissions.
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                example: ["test.user@demo.org","demo.user@demo.org"]
+                type: array
+                items:
+                  type: string
+          description: Successful operation
+        "401":
+          description: Authentication failure
+        "500":
+          description: Internal application error
+  /users/{userName}/files:
+    get:
+      description: Lists all users with ongoing submissions.
+      parameters:
+        - in: path
+          name: userName
+          schema:
+            type: string
+          required: true
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/FileInfo"
+          description: Successful operation
+        "400":
+          description: Bad username
+        "401":
+          description: Authentication failure
+        "500":
+          description: Internal application error
+components:
+  schemas:
+    C4ghKeyAdd:
+      type: object
+      properties: 
+        description:
+          type: string
+          example: pubkey description
+        pubkey:
+          type: string
+          example: 19GT1JNQVRgIGNhbiBiZSBzZXQgdG8gYGpzb25gIHRvIGdldCBsb2dzIGluIEpTT04gZm9ybWF0=
+    C4ghKeysList:
+      type: object
```

Comment:
This should be a list/array. So that the example return value of `/c4gh-keys/list` is an array.